final-project/modules/angular/onboarding/services/websocket.service.ts

```
@@ -0,0 +1,52 @@
+  message: string;
+  percentage: number;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WebSocketService {
+  private socket: WebSocket | null = null;
+  private progressSubject = new Subject<ImportProgress>();
+
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)